src/assets/main.css

```diff
@@ -98,6 +98,11 @@ textarea {
   height: 250px;
 }
 
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
```

Comment:
Je ne comprends pas le commentaire ;-)

Reply:
Dans certaines versions de safari, le fait d'enlever le style d'une liste via CSS, supprimait le role implicite de liste des `ul` et `ol`
Cette règle permet d'être tranquille en spécifiant `role="list"` sur les listes qu'on souhaite styliser différemment !

Reply:
Ah ok, merci. Peut-être clarifier: which tells screen readers the list still behave as a list even without default styles (si j'ai bien compris).